modules/ossm-about-cert-manager.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-cert-manager-integration-istio_{context}"]
+= About integrating Service Mesh with cert-manager and istio-csr
+//TP1 content influx. Title, etc may change.
+
+The cert-manager tool ensures the certificates are valid and up-to-date by attempting to renew certificates at a configured time before they expire.
```

Comment:
IMO you can either nix this sentence or move it into the `Red Hat OpenShift Service Mesh and cert-manager` section which talks about what cert-manager is in general.